docker/Dockerfile.ci_arm

```diff
@@ -53,10 +53,10 @@ ENV PATH /opt/sccache:$PATH
 COPY install/ubuntu_install_llvm_from_source.sh /install/ubuntu_install_llvm_from_source.sh
 RUN bash /install/ubuntu_install_llvm_from_source.sh 15.0.7 8b5fcb24b4128cf04df1b0b9410ce8b1a729cb3c544e6da885d234280dedeac6
```

Comment:
Given we will be installing LLVM 15 from apt repositories, this can be removed in future. We don't need to block this PR, but can you open an issue so that we can actually do this in a follow-up PR?

Reply:
https://github.com/apache/tvm/issues/14758 issue was created for this

Reply:
Thanks!